package.json

```diff
@@ -263,5 +265,8 @@
     "vite-plugin-svgr": "^3.2.0",
     "vite-tsconfig-paths": "~4.2.0",
     "vitest": "~0.32.0"
+  },
+  "nx": {
+    "includedScripts": []
```

Comment:
Why do we need this? I remember there was a problem with it. The `nx` parameter made nx think that the root is a project

Reply:
I think this one case when I ran `npx nx repair`.
Will remove.